grin/test/AbstractInterpretation/ExtendedSyntax/CreatedBySpec.hs

```diff
@@ -526,11 +570,11 @@ spec = do
                 z1 <- pure (CInt z0)
 
                 p0 <- case z0 of
-                  0 -> store z1
-                  1 -> pure (#undefined :: #ptr)
+                  0@_1 -> store z1
+                  1@_2 -> pure (#undefined :: #ptr)
                 n0 <- fetch p0
                 n1 <- pure (#undefined :: {CNode[#ptr]})
-                _1@(CNode p1) <- pure n1
+                _3@(CNode p1) <- pure n1
```

Comment:
This is clearly confusing. We should stick with one @ notation. Value/Pattern @ Var or Var @ Value/Pattern

Reply:
Alright, then we shall swap the variable name and the pattern for as-patterns. I added a few lines about this to #32.